.gitattributes

```diff
@@ -0,0 +1,11 @@
+*.cmd text eol=crlf
```

Comment:
Without this config, Git will checkout all files with the platform line ending (CRLF on Windows). 
Also, the `mvnw.cmd` batch script was committed with Unix line endings (LF). WIth this file, the line endings of `.cmd` files will be correct on Linux as well.

Over the years, I've tried many approaches to fix line endings in projects and that's the best solution I've found so far.

Reply:
I do not think this is the default git behavior. Do you happen to have `core.autocrlf true` somewhere in your git config? You can check that via `git config --list --show-origin`. In case you do have `core.autocrlf true` for whatever reason, then you could override it to `false` for the current project by running `git config --local core.autocrlf false` in the current project directory.

I must say I prefer not adding a file that that will have to be kept in sync with .editorconfig. I prefer .editorconfig to be the single source of truth for line endings.